raster-foundry/raster-foundry
# Remove PRs from CHANGELOG not targeting release branch

## Overview

Previously, PRs targeting all branches were being included in the `CHANGELOG`. Now, only PRs targeting the release branch are included in the `CHANGELOG`.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

### Notes

Commit messages in `develop` reference 0.36.x versions, but the most recent tag is 0.33.3. Also, the most recent version referenced in the `CHANGELOG` is 0.34.0. Do you have some tags locally that weren't pushed up?

## Testing Instructions

Execute the following command:

```bash
$ docker run -ti --rm -v "${PWD}:/changelog" -w "/changelog" "quay.io/azavea/github-changelog-generator:v1.14.3" \
      --token "${GITHUB_TOKEN}" \
      --future-release "0.34.0" \
      --no-issues \
      --no-author \
      --release-branch develop
```

Ensure that the diff below is the only change:

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index e45614b0..cd26fbaf 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Change Log
 
-## [0.34.0](https://github.com/raster-foundry/raster-foundry/tree/0.34.0) (2018-05-01)
+## [0.34.0](https://github.com/raster-foundry/raster-foundry/tree/0.34.0) (2018-05-03)
 
 [Full Changelog](https://github.com/raster-foundry/raster-foundry/compare/0.33.3...0.34.0)
```

Closes https://github.com/azavea/raster-foundry-platform/issues/307
